task/upload.go

```diff
@@ -573,14 +655,34 @@ func uploadTaskOutputLocations(tctx *TaskContext) ([]OutputName, []clients.Outpu
 	return outputNames, outputLocations, nil
 }
 
+func clipTaskOutputLocations(tctx *TaskContext) ([]OutputName, []clients.OutputLocation, error) {
+	playbackId := tctx.OutputAsset.PlaybackID
+	sourcePlaybackId := tctx.Task.Params.Clip.ClipStrategy.PlaybackId
+	sessionId := tctx.Task.Params.Clip.InputSessionID
+	outURL := tctx.OutputOSObj.URL
+	var mp4 string
+
+	clipRelPath := sourcePlaybackId + "/" + sessionId + "/clip_" + playbackId
```

Comment:
```suggestion
	clipRelPath := path.Join(sourcePlaybackId, sessionId, "clip_" + playbackId)
```

Reply:
This is also outdated because of last night changes